fabric-fluid-v1/src/main/java/net/fabricmc/fabric/api/fluids/v1/container/FluidContainer.java

```diff
@@ -0,0 +1,82 @@
+package net.fabricmc.fabric.api.fluids.v1.container;
+
+import java.util.Iterator;
+
+import net.minecraft.fluid.Fluid;
+import net.minecraft.fluid.Fluids;
+
+import net.fabricmc.fabric.Action;
+import net.fabricmc.fabric.api.fluids.v1.container.volume.FluidVolume;
+
+/**
+ * An object that stores fluid, it is assumed that all fluid containers that only hold one fluid fluid Volumes.
```

Comment:
`that all fluid containers that only hold one fluid fluid Volumes.`

:concern:

Reply:
mm yes I love writing docs at 10pm